src/schemes/boundary/dummy_particles/dummy_particles.jl

```diff
@@ -447,14 +500,46 @@ end
                                kernel_weight, particle, neighbor)
 end
 
+@inline function dynamic_pressure(boundary_density_calculator, density_neighbor, v,
+                                  v_neighbor_system, pos_diff, distance, particle, neighbor,
+                                  system, neighbor_system)
+    return zero(density_neighbor)
+end
+
+@inline function dynamic_pressure(boundary_density_calculator::BernoulliPressureExtrapolation,
+                                  density_neighbor, v, v_neighbor_system, pos_diff,
+                                  distance, particle, neighbor,
+                                  system::BoundarySystem, neighbor_system)
+    if system.ismoving[]
+        relative_velocity = current_velocity(v, system, particle) .-
+                            current_velocity(v_neighbor_system, neighbor_system, neighbor)
+        normal_velocity = dot(relative_velocity, pos_diff) / distance
+
+        return boundary_density_calculator.factor * density_neighbor *
+               dot(normal_velocity, normal_velocity) / 2
```

Comment:
This is not what you wrote in the docs, or am I missing something?

Reply:
see above this is identical to the equation